client/components/editor/Sidebar/index.vue

```diff
@@ -64,9 +66,12 @@ export default {
     selectedActivity: { type: Object, required: true },
     selectedElement: { type: Object, default: null }
   },
-  data: () => ({ selectedTab: 'browser', commentCount: 0 }),
+  data: () => ({ selectedTab: 'browser', unseenCount: 0 }),
```

Comment:
Rename to unseenCommentCount (this is sidebar root, so it is a bit unclear what you are referring to unless you add some context)